### PR TITLE
weeb-list

### PR DESCRIPTION
**Pull request by shubhra-g**
### ✅ Check List (Check all the applicable boxes) 

- [ x ] My code doesn't break any part of the project (Zero Octave-Javascript-Projects).
- [ x ] This PR does not contain plagiarized content.
- [ x ] My Addition/Changes works properly and matches the overall repo pattern.
- [ x ] The title of my pull request is a short description of the requested changes.

### 📷 Screenshots
image

### CODE REVIEW

File: Public/weeblist.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WeebList</title>
+    <link rel="stylesheet" href="weeblist.css" />
```
Comment (Astrodevil) [inline Public/weeblist.html:9]: Your linkage are wrong for css,Js. It should be like: (../assets/css/weeblist.css)

**Comment by Astrodevil** [comment]
Please update your project in alphabetical order on cards.json and also add it on README after updating fork.

**Comment by Astrodevil** [comment]
You have unfinished task

**Comment by shubhra-g** [comment]
I have made the necessary changes

**Comment by shubhra-g** [comment]
i will add the file by tomorrow

**Comment by Astrodevil** [comment]
Last few days remaining, Complete your work till 31st.

**Comment by Astrodevil** [comment]
@shubhra-g  You still have time till evening. After that I will be closing this PR.

**Pull request closed without merging**